content/docs/buildpack-author-guide/create-buildpack/adding-bill-of-materials.md

```diff
@@ -14,14 +14,14 @@ One of the benefits of buildpacks is they can also populate the app image with m
 
 You can find all of this information using `pack` via its `inspect-image` command.
 
-
 <!-- test:exec -->
 ```bash
 pack inspect-image test-ruby-app
 ```
+<!--+ "{{execute}}" +-->
```

Comment:
```suggestion
<!--+- "{{execute}}" +-->
```

this and others would need to strip whitespaces

Reply:
I've normalized all the `<!--+ "{{execute}}" +-->` as `<!--+- "{{execute}}"+-->` and `<!--+ "{{open}}" +-->` as `<!--+"{{open}}"+-->`.